panels/class-debug-bar-deprecated.php

```diff
@@ -129,9 +129,11 @@ function deprecated_argument_run( $function, $message, $version ) {
 		$location = $backtrace[ $bt ]['file'] . ':' . $backtrace[ $bt ]['line'];
 
 		if ( ! is_null( $message ) ) {
-			$message = sprintf( __( '%1$s was called with an argument that is <strong>deprecated</strong> since version %2$s! %3$s' ), $function, $version, $message );
+			/* translators: %1$s is a function name, %2$s a version number, %3$s a message regarding the change. */
+			$message = sprintf( __( '%1$s was called with an argument that is <strong>deprecated</strong> since version %2$s! %3$s', 'debug-bar' ), $function, $version, $message );
 		} else {
-			$message = sprintf( __( '%1$s was called with an argument that is <strong>deprecated</strong> since version %2$s with no alternative available.' ), $function, $version );
+			/* translators: %1$s is a function name, %2$s a version number. */
+			$message = sprintf( __( '%1$s was called with an argument that is <strong>deprecated</strong> since version %2$s with no alternative available.', 'debug-bar' ), $function, $version );
```

Comment:
Nah, I made sure they wouldn't ;-) And just in case they would anyway, I'll happily rebase the other one once one of the two has been merged ;-)